client/src/components/workspace-description/index.js

```diff
@@ -35,17 +34,13 @@ const WorkspaceDescription = () => {
     descriptionMarkup = <P>{description}</P>
   }
 
-  if (headline.length && description.length) {
-    return (
-      <div>
-        <GlobalStyles />
-        <H2>{headline}</H2>
-        {descriptionMarkup}
-      </div>
-    );
-  } else {
-    return <></>;
-  }
```

Comment:
curious why you are removing conditional rendering logic if there isn't a default loading state.. I guess it doesn't really matter of empty html containers load?  might not be seeing full picture here

Reply:
My reasoning is the component doesn't "break" if the service doesn't respond, and an empty element is the only side effect on the DOM. And I want the headline to still show if the location isn't there.

There probably is a cleaner way to handle this, but this was the easiest to write and read.